app/views/application/_header.html.erb

```diff
@@ -1,18 +1,25 @@
 <header class="usa-header usa-header--basic">
-  <div class="usa-nav-container">
-    <div class="usa-navbar">
-      <div class="usa-logo" id="basic-logo">
+  <div class="grid-container">
+    <div class="grid-row">
+      <div class="usa-logo grid-col-6" id="basic-logo">
         <em class="usa-logo__text">
-          <a href="javascript:void(0)" title="OHS Complaint Tracker">
             OHS Complaint Tracker
-          </a>
         </em>
       </div>
-      <% if user_signed_in? %>
-        <%= link_to "Logout #{current_user["uid"]}", logout_path, class: "usa-button", data: {turbolinks: false} %>
-      <% else %>
-        <%= button_to "Login", "/auth/hses", class: "usa-button" %>
-      <% end %>
+      <div class="grid-row grid-col-6 grid-gap flex-align-center flex-justify-end">
```

Comment:
[question] I thought that grid and flexbox were two approaches to solving layout problems, and that they weren't really meant to be mixed together. Are there `grid-align-center grid-justify-end` classes? I know those types of things exist in tailwind css but I'm not sure about uswds.

Reply:
they are USWDS classes.
https://designsystem.digital.gov/utilities/flex/

The examples all use `grid-row` and `grid-col` classes along with the flex classes.

I did do some experimenting to try to remove the `grid-col-6` class but in terms of the design layout, it seemed much cleaner to group the different elements by the column-width and then flex within that.

Reply:
Huh, yeah I don't see anything about alignment or justify in the grid docs, so I guess this is the preferred way to do it in USWDS.